### ISSUE TITLE
Feature request: logger should write to stdout directly

### ISSUE THREAD

**Issue by olanb7** [aws-powertools/powertools-lambda-typescript#747]
## Logger should write to stdout, and not use console

**Problem statement**

The powertools logger is currently outputting it's logs via `console` [Logger.ts#L451](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/8aab29900c5fac8eb625eb747acbc23ceac8f6ba/packages/logger/src/Logger.ts#L451). Lambda appears to use a patched version of `console` which prefixes the timestamp, requestId and log level. This means that a powertools logged event is output to stdout as below, which is clearly not JSON:

```
2022-04-08T10:58:44.811Z	b6511ee9-4873-404e-b60c-a23cb45d3ff2	INFO {"cold_start":false,"function_arn":"arn:aws:lambda:eu-west-2:12345:function:test","function_memory_size":1024,"function_name":"test","function_request_id":"b6511ee9-4873-404e-b60c-a23cb45d3ff2","level":"INFO","message":"test message","service":"test","timestamp":"2022-04-08T10:58:44.811Z"}
```

Cloudwatch handles this OK, but other tools like the [newrelic](https://github.com/newrelic/newrelic-lambda-extension) and [honeycomb](https://docs.honeycomb.io/getting-data-in/integrations/aws/aws-lambda/#javascript) extensions work under the assumption that JSON logs are being sent to stdout. Honeycomb even [warns against the use of console.log in their documentation](https://docs.honeycomb.io/getting-data-in/integrations/aws/aws-lambda/#javascript).

Following the honeycomb suggestion of using `bunyan` (which does write to `stdout`) I have validated (as has @dreamorosi on slack) that these prefixes can be avoided by bypassing `console.log`. 

The information in these prefixes is also duplicated already in the JSON message itself, so there is no loss of querying (although some may prefer to keep these prefixes so that cloudwatch queries aren't impacted. 

**Summary of the feature**

Update the logger to avoid the patched console.log by writing logs to stdout. This will ensure that valid JSON messages can be intercepted by lambda layers and easily forwarded on to other observability tools.

**Benefits for you and the wider AWS community**

This is a blocker right now to anyone using centralised logging tools like NewRelic or Honeycomb, for whom structured logs are critical to the maintenance and support.

**Describe alternatives you've considered**

The only alternative is to use a different logger like [pino](https://github.com/pinojs/pino) or [bunyan](https://github.com/trentm/node-bunyan).

**Additional context**

I have not been able to find the source for the lambda logger internals (presumably closed source) but have verified that it must be adding the prefixes by bypassing.

### Related issues, RFCs

[#646](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/646)
[#lambda-powertools slack discussion](https://awsdevelopers.slack.com/archives/C01A6KK4UFK/p1649340843599199)

**Comment by saragerion** [comment]
Thanks a lot of opening this issue. We want developers to be able to send their logs to external centralised monitoring tools, and for that they need to be parsed correctly as valid JSON not only by CloudWatch.

As part of the unit of work of this issue we could consider also adding an E2E tests that validates the output of the logs as valid JSON.

**Comment by dreamorosi** [comment]
We were able to come up with a fix and avoid having the prefix on the logs by overriding the global `Console` object. 

I'm working on a PR to fix this and have already applied the change, but it'll take me a while to adapt our tests to reflect the changes.

You can track progress in #748.

**Comment by dreamorosi** [comment]
The change has been merged in #748 and will be released in the next version.
